### ISSUE TITLE
[HW4] Not Found error 관련 질문

### ISSUE THREAD

**Issue by kslim99** [swsnu/swppfall2020#172]
조교님 안녕하세요, 
404 에러의 적용 범위에 대해 질문 드립니다.
문서에는 For all requests about non-existing article and comment, response with 404 라고 되어 있는데,
이 경우 article이 하나도 생성되지 않은 상태에서 /api/article에 GET request를 보낸다면 이 때에도 404 에러를 리턴하면 된다고 이해했습니다. 
제가 제대로 이해한 것이 맞나요?

**Comment by kooyunmo** [comment]
해당 부분의 설명은 존재하지 않는 특정 article이나 comment에 대해서 request를 보내는 경우에 해당합니다. 따라서 전체 article들의 리스트를 얻기 위한 `GET /api/article` request의 경우엔, article이 하나도 없는 상황이라면 404로 처리하는 것이 아니라 아무 article 정보도 없는 상태의 Json response를 리턴해야 합니다.

**Comment by kslim99** [comment]
넵 감사합니다!

**Comment by kooyunmo** [comment]
네 그렇습니다. article이 존재하지 않으면 404를 리턴하고, article은 존재하지만 그에 해당하는 comment는 하나도 없는 경우엔 비어있는 json response를 리턴하면 됩니다.

**Comment by kslim99** [comment]
넵 감사합니다!
